Sources/Interaction/Style/InteractorStyleImage/index.js

```diff
@@ -223,29 +223,19 @@ function vtkInteractorStyleImage(publicAPI, model) {
     model.currentImageNumber = i;
 
     function propMatch(j, prop, targetIndex) {
-      if (
-        prop.isA('vtkImageSlice') &&
```

Comment:
was the bug due to that specific line, now moved to l231?

Reply:
That's right. 
Relying on indexes works only if the number of props is stable. There was an issue when adding multiple non-vtkImageSlice props(e.g. widgets). Those were messing up with the index, hence targetIndex could not be found.
Solution is to filter out all non vtkImageSlice first.